apisix/cli/schema.lua

```diff
@@ -208,6 +208,25 @@ local config_schema = {
                     properties = {
                         ssl_trusted_certificate = {
                             type = "string",
+                        },
+                        listen = {
+                            type = "array",
+                            items = {
+                                type = "object",
+                                properties = {
```

Comment:
Do we need to use `required` for some fields?

Reply:
done

Reply:
many test cases doesn't have sll config, so ignore `required `

Reply:
Yes, you are right. The ip and port are not required in the code. I made a mistake.